src/allmydata/immutable/downloader/finder.py

```diff
@@ -63,6 +63,7 @@ def start_finding_servers(self):
         if not self._started:
             si = self.verifycap.storage_index
             servers = self._storage_broker.get_servers_for_psi(si)
+            servers.sort(key=lambda s: s.get_serverid())
```

Comment:
I agree this should be removed; it was in the original mark berger branch and I don't know why/what it's supposed to accomplish...